src/app/lib/config/services/afrique.js

```diff
@@ -139,6 +139,7 @@ export const service = {
         },
       },
       media: {
+        noJs: '',
```

Comment:
I think we normally have english in these configs until we get translations?  This is fine though 👍

Reply:
Ah ok. We can follow that convention when we come to plumbing the no js messages into the media player.